server/resource/caldav_mail.go

```diff
@@ -0,0 +1,110 @@
+package resource
+
+
+import (
+	"bytes"
+	"crypto/tls"
+	"encoding/base64"
+	"log"
+	"net/smtp"
+	"text/template"
+)
+
+type Mail struct {
+	Address  string `yaml:"address"`
+	Username string `yaml:"username"`
+	Password string `yaml:"password"`
+	From     string `yaml:"from"`
+}
+
+var caldavTemplate = string(`Subject: {{ .Subject }}
+To: {{ .User -}} <{{ .Email -}}>
+From: daptin <{{ .NoReply -}}>
+MIME-Version: 1.0
+Content-Type: multipart/mixed; boundary="0000"
+--0000
+Content-Type: text/calendar; charset="UTF-8"; method=REQUEST
+Content-Transfer-Encoding: 7bit
+{{ .Content }}
+--0000
+Content-Type: application/ics; name="invite.ics"
+Content-Disposition: attachment; filename="invite.ics"
+Content-Transfer-Encoding: base64
+{{ .Invite }}
+--0000--
+`)
+
+type maildata struct {
+	Subject string
+	Email string
+	NoReply string
+	Content string
+	Invite string
+	User string
+}
+
+func (mailer Mail) Send(name string, email string, content string, subject string) {
```

Comment:
Use the inbuild mail server to send mails, here is the helper function

https://github.com/daptin/daptin/blob/a3cb5b22a57b818d4e49aaadc9d1addf7516add8/server/resource/action_mail_send.go